templates/WAM_APP_FRED/test_map_layout.html

```diff
@@ -176,7 +176,11 @@ <h4>Weather Data</h4>
 
                   <!-- Third panel content -->
                   <div class="tabs-panel" id="panel-feedinlib">
-                    test
+                    <p>
+                      Click on a region displayed on the map. Click the now visible datapoint displayed 
```

Comment:
```suggestion
                      Select a variable and a time span, then click on a region highlighted on the map. The data will be displayed on a graph inside a popup.
```

The popup should open automatically : #91